test/kitchen/site-cookbooks/dd-agent-step-by-step/recipes/default.rb

```diff
@@ -40,6 +40,7 @@
       gpgcheck=1
       repo_gpgcheck=0
       gpgkey=#{protocol}://yum.datadoghq.com/DATADOG_RPM_KEY.public
+             #{protocol}://yum.datadoghq.com/DATADOG_RPM_KEY_E09422B3.public
```

Comment:
Just a note to reviewers, this is indeed the correct way to specify multiple GPG keys for a yum repository.